javascript/data.js

```diff
@@ -7,6 +7,7 @@ const steak = [
   "cook steak for 30-60 seconds per side using a spoon to baste with butter",
   "rest for 10 minutes",
   "enjoy",
+  "steak is ready",
```

Comment:
That's not the approach ;) You should be adding additional command with JS.

document.querySelector("#steak").innerHTML += `<li>Steak is ready!</li>`;